config/mainnet/config.json

```diff
@@ -1,23 +1,25 @@
 {
-  "AlonzoGenesisFile": "/config/alonzo-genesis.json",
+  "AlonzoGenesisFile": "/current/alonzo-genesis.json",
```

Comment:
Could you rename this in every config file back to config. Otherwise all of our other files can't find this.
And current is from my point of view not the right name for this directory. That's why I changed it to "config"